src/main/java/db/Storage.java

```diff
@@ -0,0 +1,16 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
I'd recommend making it private and creating a separate getter method if you need to get all entries